CMakeLists.txt

```diff
@@ -140,37 +139,23 @@ function (config_without_llvm)
 endfunction ()
 
 function (config_with_llvm)
-  execute_process(COMMAND sh -c "${LLVM_PATH}/bin/clang --version"
-     RESULT_VARIABLE clang_version_status
-     OUTPUT_VARIABLE CLANG_VERSION_OUTPUT)
-
-  set(CLANG_KNOWN_GIT_COMMIT_HASH "d3173f4ab61c17337908eb7df3f1c515ddcd428c")
```

Comment:
I think we should keep this check in place with the new hash used in the submodule. Then we don't have users accidentally running submodule update and then the build fails.

Reply:
Yes, done.